src/live-engage.android.ts

```diff
@@ -85,8 +104,8 @@ export class LiveEngage implements CommonLiveEngage {
         }
 
         // only try hideConversation() if foregroundActivity is a ConversationActivity or it will crash
-        if (application.android.foregroundActivity instanceof com.liveperson.infra.messaging_ui.ConversationActivity) {
-            com.liveperson.infra.messaging_ui.MessagingUIFactory.getInstance().hideConversation(application.android.foregroundActivity);
+        if (application.android.foregroundActivity instanceof ConversationActivity) {
+            LivePerson.hideConversation(application.android.foregroundActivity as any);
```

Comment:
Can you cast it to a ConversationActivity or foregroundActivity instead of any?

Reply:
Tried that, but it won't accept.